tests/appsec/suitespec.yml

```diff
@@ -71,23 +71,6 @@ suites:
       - tests/appsec/iast_aggregated_memcheck/*
     runner: hatch
     timeout: 50m
-#  appsec_iast_packages:
```

Comment:
we need to keep this, can delete the env, cache, and python_versions, just make it look like a normal riot suitespec definition